pgrx-sql-entity-graph/src/control_file.rs

```diff
@@ -86,6 +87,10 @@ impl ControlFile {
                 .ok_or(ControlFileError::MissingField { field: "superuser" })?
                 == &"true",
             schema: temp.get("schema").map(|v| v.to_string()),
+            trusted: temp
+                .get("trusted")
+                .ok_or(ControlFileError::MissingField { field: "trusted" })?
+                == &"false",
```

Comment:
I don't think we should require this parameter to be set. We should simply treat the absent case as equivalent to false, since it de facto is.

Reply:
Doing that would also fix the tests.

Reply:
Thanks. This field is optional now.

Reply:
whoops. accidentally out of budget.